client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,84 @@
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+import YelpLogo from "../../../assets/images/yelp-logo.png";
+import RedirectButton from "../../../components/RedirectButton";
+
+const classes = {
+  previewContainer: {
+    height: "160px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "100px",
+    marginLeft: "1rem",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "10px",
+  },
+  nameContainer: {
+    fontSize: "1.5rem",
+    textDecoration: "none",
+    color: "black",
+  },
+  categoriesContainer: {
+    marginTop: "4.8rem",
+  },
+  exitContainer: {
+    marginLeft: "6rem",
+  },
+  yelpButtonContainer: {
+    marginTop: "3.2rem",
+  },
+};
+
 function RestaurantPreview({ restaurant }) {
+  const restaurantPage = "/restaurant?yelp_id=" + restaurant.yelp_id;
+  const restaurantAddress =
+    restaurant.address.address1 +
+    ", " +
+    restaurant.address.state +
+    ", " +
+    restaurant.address.zip_code;
```

Comment:
Nit: I see a bunch of `restaurant.address` here. Destructuring the address + state + zip_code from it would prob make it cleaner

```js
const { address1, state, zip } = restaurant.address;
```

Reply:
tip: you can destructure and rename like so:
```
const obj = { some_key: 10 };
const { some_key: renamedKey } = obj;
console.log(renamedKey); // 10
```

Reply:
also edge case: 
![image](https://user-images.githubusercontent.com/51217000/230526498-92f0e6b0-2d7e-4f12-bb80-e1acdb58aeb8.png)

Reply:
here's what the same restaurant looks like on the card:
![image](https://user-images.githubusercontent.com/51217000/230527069-b298420c-df3e-40aa-a3a9-a9e4d5b198f5.png)